src/the-bureau/about-deputy-director/index.html

```diff
@@ -16,28 +16,24 @@
+            Deputy Director Meredith Fuchs
         </h1>
         <p class="h3">
-            Steven L. Antonakes currently serves as the Deputy Director as well as the Associate Director for Supervision, Enforcement, and Fair Lending at the Consumer Financial Protection Bureau (CFPB).
+            Meredith Fuchs currently serves as the Acting Deputy Director as well as the General Counsel at the Consumer Financial Protection Bureau (CFPB).
         </p>
+        <figure class="media float-media rwd-crop">
+            <img class="media_image media_image__150 rwd-crop_item"
+                 src="/static/img/deputy-director-fuchs-300x400.jpg"
+                 alt="Portrait of Deputy Director Meredith Fuchs">
+        </figure>
         <p>
-            <figure class="float-media rwd-crop">
-                <img class="rwd-crop_item"
-                     src="/static/img/201405_cfpb_photo_steve-antonakes_low-res.jpg"
-                     width="155"
-                     alt="Portrait of Deputy Director Steven Antonakes">
-            </figure>
-            He is responsible for the supervision of all banks and non-banks under the CFPB’s jurisdiction and the enforcement of federal consumer protection and fair lending laws. Mr. Antonakes previously served as the CFPB’s Assistant Director of Large Bank Supervision and was charged with establishing the consumer protection supervision program for the largest financial institutions in the country which collectively account for over 75 percent of the nation’s banking assets.
+            Meredith Fuchs, who is serving as Acting Deputy Director, is also the General Counsel of the CFPB. She joined the Bureau in 2011 as Principal Deputy General Counsel before serving as Chief of Staff to CFPB Director Richard Cordray.
         </p>
         <p>
-            Mr. Antonakes began his professional career as an entry level bank examiner with the Commonwealth of Massachusetts Division of Banks in 1990. He later served in numerous managerial capacities before being appointed by successive Governors to serve as the Commissioner of Banks from December 2003 until November 2010, becoming only the second career bank examiner to ever serve in that capacity.
+            Prior to joining the CFPB, she served as Chief Investigative Counsel of the United States House of Representatives Committee on Energy and Commerce. Previously, Ms. Fuchs held positions as Vice President and General Counsel of the National Security Archive at George Washington University, a litigation partner in private practice, the Supreme Court Assistance Project Fellow at the Public Citizen Litigation Group, and an officer on the D.C. Bar Board of Governors. She is the recipient of the American Library Association’s James Madison Award. Ms. Fuchs served as a law clerk for Judge Patricia M. Wald on the D.C. Circuit Court of Appeals and Judge Paul L. Friedman on the United States District Court for the District of Columbia.
```

Comment:
Added line breaks in https://github.com/cfpb/cfgov-refresh/commit/ecc5165e5d81e995ff8d7ceb024c6c32d17ea5a6